.github/workflows/publish-autodoc.yml

```
@@ -16,8 +16,8 @@ jobs:
       - uses: eclipse-edc/.github/.github/actions/setup-build@main
 
       - name: Override version if input is set
-        if: "${{ github.event.inputs.version != '' }}"
-        run: sed -i "s/version=.*/version=${{ github.event.inputs.version }}/g" gradle.properties
+        if: "${{ inputs.version != '' }}"
```

how about:
```suggestion
        if: "${{ inputs.version != '' || github.event.inputs.version != '' }}"
```
that way it'll fallback to the event's version input if the workflow input is not set

---

I think it's not needed, this flow can only start from a workflow_call, so the `inputs.version` is the real source of trouth. 
Checking the original event would add additional complexity because then we'll need to chose which one will be used in the `sed` comment.